install-on-TPU-v4.sh

```diff
@@ -0,0 +1,24 @@
+sudo apt remove unattended-upgrades
```

Comment:
why do you remove unattended-upgrades?

Reply:
They kicked off twice, each after a sudo apt update and kept the TPU VM stuck for more than 90 minutes before I decided to just kill them. I consider the lifetime of a TPU VM to be short and the VM not to be exposed to the outside world. Hence, I think getting a stuck (costly) VM due to some potentially non-critical updates seems worse than not having this service and instead doing updates as per your own schedule.

Reply:
I understand the issue, but I think that depends on the distribution you are using (I haven't experienced it so far), not necessarily related to `optimum-tpu`, that should provide tools for machine learning on TPUs. Please remove this command, consider doing the command when you are setting up your machine, before using `optimum-tpu`.